### DISCUSSION TITLE
Mock E2E Testing

### DISCUSSION THREAD

**Discussion by hussmartinez** [wevm/wagmi#4168] [Q&A]
In the previous version I could mock a wallet,
now we have the mock from connectors, but is not properly explained the usage.

I have been battling for a couple of days trying to make it work with rainbowkit, but the mocked connector doesn't appear.

Any suggestions?

**Comment by dikobay** [comment]
Hello @hussedev. Did you find the solution?

**Comment by dikobay** [comment]
Hello @hussedev. Did you find the solution?

**Reply by hussmartinez** [reply]
No I didn't find it regarding this.
But manage to get synpress up and running and it has wallet mock

https://github.com/gitcoinco/retrofunding/tree/test%2Fsynpress/tooling%2Fe2e-voting%2Fwallet-setup
